gnovm/memfile.go

```diff
@@ -87,6 +89,28 @@ func (mempkg *MemPackage) Validate() error {
 		prev = file.Name
 	}
 
+	pIndex := strings.Index(mempkg.Path, "/p/")
```

Comment:
I don't think the MemFile should have a notion of parsing go code at this stage.

Can we have this happen in `Preprocess` instead? Also, we should reuse the `IsRealmPath` function.